docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,428 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+## Prerequisites
+- Teleport v14 or later.
+- For Self-Hosted Teleport the [AWS Athena Backend](../reference/backends.mdx#athena) is required.
+
+
+### Configuration
+
+<Tabs>
+<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+
+To enable Access Monitoring you need to update the Teleport Auth Service configuration:
+```yaml
+auth_service:
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-log-example/report_results
+```
+
+</TabItem>
+<TabItem scope={["cloud","team"]} label="Cloud-Hosted">
+
+Teleport Access Monitoring is enabled by default for all Teleport Cloud users.
+
+</TabItem>
+</Tabs>
+
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `security_report` and `audit_query` resources.
+The preset `auditor` role has these permissions by default. Alternatively, you can create a custom role with these permissions:
+```yaml
+kind: role
+metadata:
+  name: my-role
+spec:
+  allow:
+    rules:
+    - resources:
+      - security_report
+      - audit_query
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+Users can write custom SQL queries for these views to build custom reports akin to querying a relational database.
+
+Within the Query Editor, users have access to a number of SQL views representing audit events captured by Teleport.
+Below is a list of the available SQL views:
+
+```
```

Comment:
Can this be made a collapsible block?

Reply:
We do support it. You can use "details" block with the "closed" tag:

```
<Details title="Available SQL views" closed>
...
</Details>
```

Reply:
Thanks.